docs/basic-usage/pusher.md

```diff
@@ -40,9 +40,9 @@ To do this, you should add the `host` and `port` configuration key to your `conf
     'options' => [
         'cluster' => env('PUSHER_APP_CLUSTER'),
         'encrypted' => true,
-        'host' => '127.0.0.1',
-        'port' => 6001,
-        'scheme' => 'http',
+        'host' => env('PUSHER_APP_HOST', '127.0.0.1'),
+        'port' => env('PUSHER_APP_PORT', 6001),
```

Comment:
Should add the path option in here too. It is how we handle things locally using the same  local host for sockets
`
     'path' => env('PUSHER_PATH'),
`